### PR TITLE
📝(git) add guideline for the commit message body

### PR DESCRIPTION
**Pull request by sampaccoud**
### Purpose

One of the most common mistake we observe from new committers is on the content of their commit messages.

### Proposal

Ironically, the problem is best explained by the diff of this commit . ^^

### CODE REVIEW

**Review by madmatah — APPROVED** [review]

File: git.md [hunk 1]
```diff
@@ -30,14 +30,58 @@ And the **scope** should point to the django application or stack component that
...
+moment of committing the changes.
+
+You can also imagine someone angry chasing you in 6 months and asking:
+
+Why on earth did you do this ??
+
+You surely wouldn't answer with a list of what you did. You would try to explain and
+justify what you did 6 months ago. The problem is that, at that
+time, you will have forgotten about all these details... and your changes may
+be misunderstood. This will lead to tensions and criticism on the quality of your work, or worse to reverting an important change for lack of understanding the implications.
```
Comment (lunika) [inline git.md:59]: break this sentence in 2 lines 😁

**Review by lunika — APPROVED** [review]

File: git.md [hunk 2]
```diff
@@ -30,14 +30,58 @@ And the **scope** should point to the django application or stack component that
...
 * use the imperative, present tense: "change" not "changed" nor "changes"
 * don't capitalize first letter
 * no dot \(.\) at the end
 
+#### Body
+
 Just as in the subject, use the imperative, present tense for the **body**: "change" not "changed" nor "changes". The body should include the motivation for the change and contrast this with the previous behavior.
 
+In other words, you should explain WHY you do what you do, and what was the context
+that lead you to do it the way you do it. It should not be a list of the things
```
Comment (jmaupetit) [inline git.md:46]:
```suggestion
that led you to do it the way you do it. It should not be a list of the things
```

File: git.md [hunk 3]
```diff
@@ -30,14 +30,58 @@ And the **scope** should point to the django application or stack component that
...
+you change, because that's what `git diff` is here for.
+
+Try to see the commit message as a "brain dump" of the state of your mind at the
+moment of committing the changes.
+
+You can also imagine someone angry chasing you in 6 months and asking:
+
+Why on earth did you do this ??
```
Comment (jmaupetit) [inline git.md:54]:
```suggestion
> Why on earth did you do this ??
```

**Review by jmaupetit — APPROVED** [review]

File: git.md (hunk 1 above)
Comment (sampaccoud) [reply git.md:59]: Good idea :sweat_smile:

**Pull request merged**